desk/app/heap.hoon

```diff
@@ -171,6 +171,7 @@
   =+  !<([old=versioned-state cool=epic:e] vase)
   =.  state  old
   =.  cor  restore-missing-subs
+  =.  cor  leave-channels-from-left-groups
```

Comment:
because we have to scry for groups you'll need to have this agent poke itself to run the channel leaves, because scries can't be done in on-load

Reply:
Huh, in testing it does seem to actually do the scry, though.

Reply:
Switched to using pokes.